webrtc.html

```diff
@@ -8335,6 +8362,28 @@ <h3><dfn>RTCIceTransport</dfn> Interface</h3>
           <code><a>RTCPeerConnection</a></code> object associated with this
           <a>ICE Agent</a>.</p>
         </li>
+        <li>
+          <p>If <var>connection</var>'s <a>[[\IsClosed]]</a> slot is
+          <code>true</code>, abort these steps.</p>
+        </li>
+        <li>
+          <p>If either <var>connection</var>'s <a>[[\PendingLocalDescription]]</a>
+          or <a>[\CurrentLocalDescription]]</a> internal slots are not
```

Comment:
This would apply to either SLD(answer) or SRD(answer), I think you only want SLD(answer)?